### ISSUE TITLE
Dynamic entity does not work

### ISSUE THREAD

**Issue by Tinathnath** [JDetmar/NLog.Extensions.AzureStorage#38]
Hi,
First, thank you for this Nuget, it is very helpfull :)

I'm using the Table Storage target but I'm facing an issue with Dynamic Entity, it keeps using the default NLog Entity.

Here's my config:

```xml
    <target name="documentHistory"
        xsi:type="AzureTableStorage"
        ConnectionStringKey="AzureStorage"
        tableName="fileoperationshistory" >
      <contextproperty name="Level" layout="${level}" />
      <contextproperty name="Message" layout="${message:raw=true}" />
      <contextproperty name="Document" layout="${event-properties:document}" />
      <contextproperty name="Action" layout="${event-properties:action}" />
      <contextproperty name="User" layout="${event-properties:user}" />
      <contextproperty name="FullMessage" layout="${message}" />
      <contextproperty name="Details" layout="${event-properties:details}" />
    </target>
```

And here's the Table generated structure:
![image](https://user-images.githubusercontent.com/3273299/63705149-dc33a880-c82c-11e9-9576-088997304fe1.png)

Am I doing something wrong ? Or is it an actual bug ?

Thanks !

**Comment by snakefoot** [comment]
Have you tried updating to ver. 1.1.4 ?

https://www.nuget.org/packages/NLog.Extensions.AzureStorage/

It includes this fix: #35

**Comment by Tinathnath** [comment]
My bad, I just came back from holiday and didn't see you pushed a new release.
v1.1.4 does fix the issue.
Thanks !
